discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
```

Comment:
What kind of API documentation this lib has. Ultimately I decided for the following set of values: `automatic` if it's automatically generated from the code, `manual` if it's manually updated, `none` if there is none. For me, they were the clearest.